modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/layer/SymbolMBLayer.java

```diff
@@ -1663,7 +1676,7 @@ public Expression textTranslateAnchor() {
         List<Symbolizer> symbolizers = new ArrayList<Symbolizer>();
 
         LabelPlacement labelPlacement;
-
+        Boolean followLine = false;
```

Comment:
As far as I can tell, this doesn't do anything, whenever we have a linePlacement, followline is always set to true.